Clinical-Genomics/scout
# Remove MT variants with AD alt=0 from MT report

fix #1278

**How to test**:
1. Use the scout-demo database and modify the following het variant: {_id: '123c73baa49053a95b8f6058164a32e6'} so that it becomes homozygous (0/0) and with 0 reads in the in one of the 3 individuals: 
![image](https://user-images.githubusercontent.com/28093618/62676554-9f059480-b9ab-11e9-9cbe-2f10f46a32e1.png)

1. On scout demo it will look like this:
![image](https://user-images.githubusercontent.com/28093618/62676534-81382f80-b9ab-11e9-8df6-eb4fceee5cbb.png)

1. Using the master branch (not this one) make sure that when you export the MT record and open the specific excel sheet for that individual there will be 3 lines. One of these lines should have the variant above with AD alternative = 0

1. Checkout to this branch and re-export the the MT report.


**Expected outcome**:
The line with AD alternative=0 should disappear:
![image](https://user-images.githubusercontent.com/28093618/62676729-13403800-b9ac-11e9-81d0-0a4aa7027ac2.png)


**Review:**
